src/interfaces/filing-interfaces/alterationFiling-interface.ts

```diff
@@ -0,0 +1,18 @@
+import { AlterationIF } from '@/interfaces'
+
+export interface AlterationFilingIF {
+  header: {
+    name: string
+    certifiedBy: string
+    date: string
+    filingId?: number // Optional as this is not required when building a filing - causes an error for new filings
+    folioNumber?: string // Optional to the user and only displayed for certain account types
+  }
```

Comment:
No change needed now, but in future we will need to add staff payment fields in here.

Reply:
right, good catch, thanks!